browser/alternate_private_search_engine_controller.cc

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/alternate_private_search_engine_controller.h"
+
+#include "base/strings/utf_string_conversions.h"
+#include "brave/common/pref_names.h"
+#include "chrome/browser/profiles/profile.h"
+#include "chrome/browser/search_engines/template_url_service_factory.h"
+#include "components/prefs/pref_service.h"
+#include "components/search_engines/template_url_service.h"
+
+namespace {
+
+TemplateURLData GetPrivateSearchEngineData() {
+  TemplateURLData private_search_engine_data;
+  private_search_engine_data.SetShortName(base::ASCIIToUTF16("DuckDuckGo"));
+  private_search_engine_data.SetKeyword(base::ASCIIToUTF16("duckduckgo.com"));
+  private_search_engine_data.SetURL(
+      "https://duckduckgo.com/?q={searchTerms}&atb=v128-6_e");
```

Comment:
I won't block on this but please fix in a follow up.
what is this for? `&atb=v128-6_e`
Please remove that and instead add the brave referral code of `t=brave`

Reply:
Done.